packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -533,15 +531,15 @@ export class MgtPerson extends MgtTemplatedComponent {
   @state() private _personCardShouldRender: boolean;
   @state() private _hasLoadedPersonCard = false;
 
-  private _personDetailsInternal: IDynamicPerson;
-  private _personDetails: IDynamicPerson;
-  private _fallbackDetails: IDynamicPerson;
-  private _personImage: string;
-  private _personPresence: Presence;
-  private _personQuery: string;
-  private _userId: string;
-  private _usage: string;
-  private _avatarType: avatarType;
+  @state() private _personDetailsInternal: IDynamicPerson;
```

Comment:
I'm not sure that we should be making the backing fields of our `@property` decorated fields into `@state` fields perhaps we should move the property decorators to the setters, thus allowing us to remove the explicit calls to `requestUpdate` and reduce the amount of work that lit is needing to do and state to track.

Reply:
Yes, Implementing this change in PR #2861

Reply:
TBH I'd like to see the state decorators removed here as I don't think we need them for this to work.